LocalSettings.php

```diff
@@ -3385,10 +3380,14 @@
 
 	// MultiBoilerplate
 	'wgMultiBoilerplateDisplaySpecialPage' => [
-		'wmgUseMultiBoilerplate' => true,
```

Comment:
This should still be wmgUseMultiBoilerplate

Reply:
Did the change I make do it, or do I need to change all of them?